src/glb-director/tests/test_director_classify_ranges_v4.py

```diff
@@ -19,9 +19,10 @@
 from scapy.all import Ether, IP, IPv6, Packet, UDP, TCP, ICMP
 from nose.tools import assert_equals
 from nose.plugins.attrib import attr
+from nose.plugins.skip import SkipTest
 import socket, struct, time
 
-@attr(director_type='dpdk')
+#@attr(director_type='dpdk')
```

Comment:
I think we can just remove this rather than comment it, I like the SkipTest approach :+1:

```suggestion
```